ttnn/cpp/ttnn/run_operation.hpp

```diff
@@ -94,7 +94,7 @@ Tensors run_with_autoformat(
     const Tensors& input_tensors,
     const OptionalConstTensors& optional_input_tensors = {},
     const OptionalTensors& optional_output_tensors = {},
-    const float pad_value = 0,
+    const std::variant<int, float> pad_value = (float)0.0,
```

Comment:
```suggestion
    const std::variant<int, float> pad_value = 0.0f,
```
is probably more idiomatic